dataset/src/main/scala/frameless/functions/NonAggregateFunctions.scala

```diff
@@ -2,7 +2,9 @@ package frameless
 package functions
 
 import frameless._
-import org.apache.spark.sql.{functions => untyped}
+import org.apache.spark.sql.{ functions => untyped }
```

Comment:
👏

Reply:
@pathikrit hate to rain on this but the formatting in frameless seems to be no spacing after curly. I'll be reverting this change. 😞